components/zendesk/sources/closed-ticket/closed-ticket.mjs

```diff
@@ -0,0 +1,52 @@
+import common from "../common.mjs";
+
+export default {
+  ...common,
+  name: "Closed Ticket",
+  key: "zendesk-closed-ticket",
+  type: "source",
+  description: "Emit new event when a ticket is closed",
+  version: "0.0.1",
```

Comment:
Could we use a [built-in deduping strategy](https://pipedream.com/docs/components/guidelines/#deduping) for this source and the other sources?

```suggestion
  version: "0.0.1",
  dedupe: "unique",
```